craft_application/models/spread.py

```diff
@@ -94,9 +95,9 @@ class SpreadSystem(SpreadBaseModel):
     workers: int | None = None
 
     @classmethod
-    def from_craft(cls, simple: CraftSpreadSystem | None) -> Self:
+    def from_craft(cls, csystem: CraftSpreadSystem | None) -> Self:
```

Comment:
Just renaming some variables since `simple` is a leftover from an old class name.